CHANGELOG.md

```diff
@@ -3,7 +3,12 @@
 *Important!*
 - The next release will be 1.0.0, which means that master will contain breaking changes.
 
-## 0.31
+## 0.31.1
+
+*Bug Fixes*
+- Fix a scoping issue of ClusterResourceDiscovery where it was not visible to kubernetes-run, causing a crash. ([#624](https://github.com/Shopify/kubernetes-deploy/pull/624))
+
+## 0.31.0
```

Comment:
`## 0.31.0 (yanked)` ? Should leave some note that we pulled this version due to a bug.

Reply:
Yeah, I didn't want to put that until we actually yank the gem itself. Perhaps `(broken, do not use)`?

Reply:
Let's just hold off then.